baseapp/baseapp.go

```diff
@@ -151,7 +150,7 @@ func NewBaseApp(
 		logger:           logger,
 		name:             name,
 		db:               db,
-		cms:              store.NewCommitMultiStore(db),
+		cms:              rootmulti.NewStore(db, logger),
```

Comment:
interesting that the NewCommitMultistore uses a NoOpLogger 🤔 
```
func NewCommitMultiStore(db dbm.DB) types.CommitMultiStore {
	return rootmulti.NewStore(db, log.NewNopLogger())
}
``` i guess should be ok